flytekit/core/python_auto_container.py

```diff
@@ -169,6 +174,13 @@ def get_container(self, settings: SerializationSettings) -> _task_model.Containe
             memory_limit=self.resources.limits.mem,
         )
 
+    @abstractmethod
+    def execute(self, **kwargs) -> Any:
+        """
+        This method will be invoked to execute the task.
+        """
+        pass
```

Comment:
what is this method for?

Reply:
to fix the linter error, we need to extend from ABC otherwise

Reply:
done